sdk/servicebus/service-bus/src/core/managementClient.ts

```diff
@@ -577,6 +588,7 @@ export class ManagementClient extends LinkEntity {
           operation: Constants.operations.scheduleMessage
         }
       };
+      const associatedLinkName = this._context.sender!.name;
```

Comment:
It is possible that a `schedlueMessages` call is made by the user when there is no underlying AMQP sender link. In that case, `this._context.sender` would be `undefined` and the above would throw an error. Which is why we checked for `this._context.sender` before accessing its name to use as associated link before the changes in this PR.